src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -43,12 +44,13 @@ public AddCommand parse(String args) throws ParseException {
         Name name = ParserUtil.parseName(argMultimap.getValue(PREFIX_NAME).get());
         Phone phone = ParserUtil.parsePhone(argMultimap.getValue(PREFIX_PHONE).get());
         Email email = ParserUtil.parseEmail(argMultimap.getValue(PREFIX_EMAIL).get());
+        Photo photo = new Photo(""); // add command does not allow adding photo straight away
```

Comment:
We may have some null initializer to make more graceful empty objects!

Reply:
So replace it with new Photo() and handle the null inside Photo instead of using an empty string?